news/2025/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer.md

```diff
@@ -0,0 +1,289 @@
+---
+layout: post
+title: "The Followpoint: Spectator, the osu!catch Mapping Pioneer"
+date: 2025-04-30 09:00:00 +0000
+series: The Followpoint
+---
+
+Have you ever looked at osu! interviews and thought to yourself: "Hey, aren't there too few mapper interviews?" Well, today is your lucky day, as we are interviewing Spectator, one of the oldest osu!catch mappers out there!
+
+![](/wiki/shared/news/2025-04-30-the-followpoint-spectator-the-osu-catch-mapping-pioneer/banner.jpg)
+
+You know how there are players, mappers and [Beatmap Nominators](/wiki/People/Beatmap_Nominators), right? And it's not exactly common for one to do two, or even all three of these activities, but [Spectator](https://osu.ppy.sh/users/702598) is exactly this kind of person! While being a top player [peaking at rank #2](https://web.archive.org/web/20120701011805/http://osu.ppy.sh:80/p/pp/?m=2) and winning multiple [osu!catch World Cups](/wiki/Tournaments/CWC), he's also a mapper with many works of his in both [Ranked](/wiki/Beatmap/Category#ranked) and [Loved](/wiki/Beatmap/Category#loved) categories. He's also a Beatmap Nominator, and not just a regular one, but an [Elite Nominator](/wiki/People/Elite_Nominators) with 145 beatmaps pushed to Ranked in 2024!
```

Comment:
a GMT as well

Reply:
Not sure how well it will work in the current intro but alrighty